package.json

```diff
@@ -40,7 +40,9 @@
     "walk-sync": "^2.0.2"
   },
   "dependencies": {
+    "@markbind/core": "^5.4.0",
```

Comment:
? why is there this change?

Reply:
Hi, this is due to importing logger as per the eslint

Reply:
Why not just use console.log ? There's no need to have this dependency introduced for the sake of a log to the console